common/perf-test-core/src/main/java/com/azure/perf/test/core/PerfStressProgram.java

```
@@ -199,35 +196,39 @@ public static void runTests(PerfStressTest<?>[] tests, boolean sync, int paralle
                 return String.format("%d\t\t%d\t\t%.2f", currentCompleted, totalCompleted, averageCompleted);
             }, true, true);
 
-        if (sync) {
-            ForkJoinPool forkJoinPool = new ForkJoinPool(parallel);
-            try {
-                forkJoinPool.submit(() -> {
-                    IntStream.range(0, parallel).parallel().forEach(i -> runLoop(tests[i], i, endNanoTime));
-                }).get();
-            } catch (InterruptedException | ExecutionException e) {
-                throw new RuntimeException(e);
+        try {
+            if (sync) {
+                ForkJoinPool forkJoinPool = new ForkJoinPool(parallel);
+                try {
+                    forkJoinPool.submit(() -> {
+                        IntStream.range(0, parallel).parallel().forEach(i -> runLoop(tests[i], i, endNanoTime));
+                    }).get();
+                } catch (InterruptedException | ExecutionException e) {
+                    throw new RuntimeException(e);
+                }
```

This can be removed now since we have a catch-all block below.